build.xml

```diff
@@ -4,10 +4,10 @@
 <project default="dist-cubrid" name="CUBRID JDBC Driver">
     <property name="cubrid-jar-file" value="JDBC-${version}-cubrid.jar"/>
     <property name="cubrid-src-jar-file" value="JDBC-${version}-cubrid-src.jar"/>
-    <property name="output" value="./output"/>
-    <property name="bin-cubrid" value="./output/bin-cubrid"/>
-    <property name="src-cubrid" value="./output/src-cubrid"/>
-    <property name="src" value="."/>
+    <property name="output" value="${basedir}/output"/>
+    <property name="bin-cubrid" value="${basedir}/output/bin-cubrid"/>
+    <property name="src-cubrid" value="${basedir}/output/src-cubrid"/>
```

Comment:
if src-cubrid and bin-cubrid should have a relative path from output, please use location="${output}/bin".